server/models/Transfer.js

```diff
@@ -180,6 +182,9 @@ class Transfer {
         // TODO: add boolean for claim in transferRepository
         claim: claimBoolean,
       });
+
+      log.info(transfer);
+      log.info('here');
```

Comment:
let's remove these. Also if at all you want to log anything, log.debug should be used

Reply:
Removed. Noted on log.debug